__test__/beer-router.test.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+// mock env
+process.env.PORT = 7000;
+process.env.MONGODB_URI = 'mongodb://localhost/testing';
+
+const faker = require('faker');
+const superagent = require('superagent');
+const Beer = require('../model/beer.js');
+const server = require('../lib/server.js');
+
+const apiURL = `http://localhost:${process.env.PORT}`;
+
+const beerMockCreate = () => {
+  return new Beer({
+    brand: faker.lorem.words(10),
+    type: faker.lorem.words(100),
+    abv: 4.2,
+  }).save();
+};
+
+describe('/api/beers', () => {
+  beforeAll(server.start);
+  afterAll(server.stop);
+  afterEach(() => Beer.remove({}));
+
+  describe('POST /api/beers', () => {
+    test('should respond with a beer and 200 status', () => {
```

Comment:
Try to add more details on when a test should pass. e.g. "should respond with a beer and 200 status when an object is successfully created"